.changeset/nice-llamas-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-styles': minor
+---
+
+Changed the info color from yellow (#fc0) to light blue (#cce4ee) according to a design update. This affects alerts and toasts as well as the background utility
```

Comment:
```suggestion
Changed the info color from yellow (#fc0) to light blue (#cce4ee) according to a design update. This affects alerts, toasts, buttons and stepper as well as the background-, border- and text-utility
```

Reply:
You probably won't change the stepper color.
If this is so, please update stepper.scss to use the yellow color again instead of the new info color.

Reply:
Extracted the change to its own ticket #884.